api-reference/10 UI Widgets/dxForm/5 Item Types/SimpleItem/editorType.md

```diff
@@ -10,10 +10,6 @@ Specifies which editor widget is used to display and edit the form item value.
 ---
 [important]If you use [DevExtreme modules](/concepts/Common/Modularity '/Documentation/Guide/Common/Modularity/'), import the editor's module when specifying this option. You can omit modules for *"dxTextBox"*, *"dxDateBox"*, *"dxCheckBox"*, and *"dxNumberBox"* because the **Form** widget imports them automatically when creating form items. 
```

Comment:
```suggestion
[important]If you use [DevExtreme modules](/concepts/Common/Modularity '/Documentation/Guide/Common/Modularity/'), import the editor's module when specifying this option. You can omit modules for *"dxTextBox"*, *"dxDateBox"*, *"dxCheckBox"*, and *"dxNumberBox"*, because the **Form** widget imports them automatically when creating form items. 
```